app/views/tables/_table_labels.html.erb

```diff
@@ -24,8 +24,37 @@
     <div class="text-danger">This table is <strong>not</strong> being run for organized play credit.</div>
 <% end %>
 <% if @table.premium? %>
-    <div class="text-warning">This is a premium table, with a cost of <%= number_to_currency(@table.price) %><% if @table.early_bird_discount? %>,
-          going up to <%= number_to_currency @table.onsite_price %> after <%= @table.prereg_ends.localtime.to_formatted_s(:short)%>
+    <div class="text-warning">This is a premium table, with a cost of <%= number_to_currency(@table.price) %>
+      <% if @table.early_bird_discount? %>,
+          going up to <%= number_to_currency @table.onsite_price %>
+          after <%= @table.prereg_ends.localtime.to_formatted_s(:short) %>
```

Comment:
Also, the format is different between the two (different use cases) so sharing wouldn't be as effective.

The buttons w/ modals were all designed to be inline, while these are all divs. 
I could play with it, though, to see how it slooks.